assets/js/components/notifications/SetupSuccessBannerNotification.js

```diff
@@ -140,11 +171,20 @@ function SetupSuccessBannerNotification() {
 	}
 
 	// Only show the connected win when the user completes setup flow.
-	const notification = getQueryParameter( 'notification' );
 	if ( ! notification || '' === notification ) {
 		return null;
 	}
 
+	if ( notification === 'authentication_success' ) {
+		if ( ! canManageOptions ) {
+			return null;
+		}
+
+		if ( slug && ! modules[ slug ]?.active ) {
+			return null;
+		}
+	}
```

Comment:
We can probably simplify this a bit 🙂 

```suggestion
	if ( notification === 'authentication_success' && ! canManageOptions ) {
		return null;
	}

	if ( notification === 'authentication_success' && slug && ! modules[ slug ]?.active ) {
		return null;
	}
```